install.sh

```diff
@@ -2842,34 +1972,75 @@ FIREEDGE_DEV_FILES="src/fireedge/src \
-
-FIREEDGE_SUNSTONE_ETC_VIEW_USER="src/fireedge/etc/sunstone/user/vm-tab.yaml \
-                                src/fireedge/etc/sunstone/user/vm-template-tab.yaml"
+                       src/fireedge/etc/sunstone/tab-manifest.yaml \
+                       src/fireedge/etc/sunstone/default-labels.yaml \
+                       src/fireedge/etc/sunstone/remotes-config.yaml"
+
+FIREEDGE_SUNSTONE_ETC_PROFILES="src/fireedge/etc/sunstone/profiles/windows_optimized.yaml \
+                                src/fireedge/etc/sunstone/profiles/base.template"
+
+FIREEDGE_SUNSTONE_ETC_VIEW="src/fireedge/etc/sunstone/views/sunstone-views.yaml"
+
+FIREEDGE_SUNSTONE_ETC_VIEW_ADMIN="src/fireedge/etc/sunstone/views/admin/vm-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/vm-template-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/vm-group-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/marketplace-app-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/vnet-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/vnet-template-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/image-tab.yaml\
+                                src/fireedge/etc/sunstone/views/admin/file-tab.yaml\
+                                src/fireedge/etc/sunstone/views/admin/sec-group-tab.yaml\
+                                src/fireedge/etc/sunstone/views/admin/backup-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/datastore-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/vdc-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/user-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/service-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/service-template-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/vrouter-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/vrouter-template-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/backupjobs-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/host-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/group-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/acl-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/cluster-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/support-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/zone-tab.yaml \
+                                src/fireedge/etc/sunstone/views/admin/marketplace-tab.yaml"
+
+FIREEDGE_SUNSTONE_ETC_VIEW_USER="src/fireedge/etc/sunstone/views/user/vm-tab.yaml \
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

FIREEDGE_SUNSTONE_ETC_VIEW_USER appears unused. Verify use (or export if used externally).